staging/src/k8s.io/apiserver/pkg/apis/apiserver/validation/validation.go

```diff
@@ -100,21 +100,33 @@ func validateJWTAuthenticator(authenticator api.JWTAuthenticator, fldPath *field
 func validateIssuer(issuer api.Issuer, fldPath *field.Path) field.ErrorList {
 	var allErrs field.ErrorList
 
-	allErrs = append(allErrs, validateURL(issuer.URL, fldPath.Child("url"))...)
+	allErrs = append(allErrs, validateIssuerURL(issuer.URL, fldPath.Child("url"))...)
```

Comment:
Where is the unique issuer check?

Reply:
They need to be unique but as of this PR we still only support 1 JWT authenticator. I've a PR open to support multiple https://github.com/kubernetes/kubernetes/pull/123431 and added the issuer unique check there.